AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -1443,6 +1443,7 @@ theguardian.com##.contributions__adblock
 !
 ! SECTION: Self-promo - Regular rules
 !
+damninteresting.com##.plea
```

Comment:
Please move it here
https://github.com/AdguardTeam/AdguardFilters/blob/f1c60ef2c07caebcc4c3fc35071e35a640bda3d6/AnnoyancesFilter/Widgets/sections/widgets.txt#L1034

Reply:
done!